fortinet-fortirecon-easm/assets.py

```diff
@@ -101,15 +101,46 @@ def get_security_insights(config, params):
     return get_multiple_records(config=config, endpoint=endpoint)
 
 
-def get_security_insights(config, params):
```

Comment:
Why this function is removed ?

Reply:
This is moved to the assets.py, I have basically moved the multiple python files of like ip, asn, domain etc to assets.py